<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>DialpadWidget</class>
 <widget class="QWidget" name="DialpadWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>369</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" >
    <widget class="QFrame" name="frameUpQComboBox" >
     <property name="minimumSize" >
      <size>
       <width>150</width>
       <height>16</height>
      </size>
     </property>
     <property name="cursor" >
      <cursor>0</cursor>
     </property>
     <property name="mouseTracking" >
      <bool>false</bool>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="minimumSize" >
      <size>
       <width>150</width>
       <height>198</height>
      </size>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" >
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="3" column="2" >
          <widget class="QToolButton" name="poundButton" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="icon" >
            <iconset resource="wengophone.qrc" >:/pics/dialpad/12_on.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QToolButton" name="oneButton" >
           <property name="maximumSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="icon" >
            <iconset resource="wengophone.qrc" >:/pics/dialpad/1_on.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QToolButton" name="starButton" >
           <property name="maximumSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="icon" >
            <iconset resource="wengophone.qrc" >:/pics/dialpad/10_on.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QToolButton" name="sevenButton" >
           <property name="maximumSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="icon" >
            <iconset resource="wengophone.qrc" >:/pics/dialpad/7_on.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QToolButton" name="twoButton" >
           <property name="maximumSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="icon" >
            <iconset resource="wengophone.qrc" >:/pics/dialpad/2_on.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QToolButton" name="threeButton" >
           <property name="maximumSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="icon" >
            <iconset resource="wengophone.qrc" >:/pics/dialpad/3_on.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QToolButton" name="sixButton" >
           <property name="maximumSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="icon" >
            <iconset resource="wengophone.qrc" >:/pics/dialpad/6_on.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QToolButton" name="fiveButton" >
           <property name="maximumSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="icon" >
            <iconset resource="wengophone.qrc" >:/pics/dialpad/5_on.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2" >
          <widget class="QToolButton" name="nineButton" >
           <property name="maximumSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="icon" >
            <iconset resource="wengophone.qrc" >:/pics/dialpad/9_on.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QToolButton" name="fourButton" >
           <property name="maximumSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="icon" >
            <iconset resource="wengophone.qrc" >:/pics/dialpad/4_on.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QToolButton" name="zeroButton" >
           <property name="maximumSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="icon" >
            <iconset resource="wengophone.qrc" >:/pics/dialpad/11_on.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QToolButton" name="eightButton" >
           <property name="maximumSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="icon" >
            <iconset resource="wengophone.qrc" >:/pics/dialpad/8_on.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>46</width>
             <height>46</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="wengophone.qrc" />
 </resources>
 <connections/>
</ui>
